app/uk/gov/hmrc/helptosavefrontend/metrics/Metrics.scala

```diff
@@ -72,6 +74,10 @@ class Metrics @Inject() (metrics: com.kenshoo.play.metrics.Metrics) {
 
   val authTimer: Timer = timer("frontend.auth-timer")
 
+  val healthCheckFailuresHistogram: Histogram = histogram("frontend.health-check.nsi.failures")
+
+  val healthCheckTimer: Timer = timer("frontend.health-check.nsi.timer")
```

Comment:
frontend.health-check  ...or ...  frontend.nsi-account-creation-time  ?
Additionally, pls can these prop names be as outlined in the Jira ticket